src/components/gameRules/gameRules.css

```diff
@@ -0,0 +1,30 @@
+.mode-rules_wrapper {
```

Comment:
why you mix dashes with underscores?

Reply:
Actually, I think it should be named "mode-rules". I used a dash to make it more readable. I got used to naming classes with BEM pattern. What name would you suggest in this case? Is camelCase good practice to name classes?